cylc/flow/platforms.py

```diff
@@ -30,6 +31,9 @@
     ('job', 'batch submit command template', [None])
 )
 
+# Regex to check whether a string is a command
+REC_COMMAND = re.compile(r'(`|\$\()\s*(.*)\s*([`)])$')
```

Comment:
Looks fine, it can be broken but only for facetious examples like:

```
1) $(foo`
2) `foo)
```

Which probs isn't an issue.

If testing for the end of the line, best to permit arbitrary whitespace:

```suggestion
REC_COMMAND = re.compile(r'(`|\$\()\s*(.*)\s*([`)])\s*$')
```

Consider testing for the start of the line too:

```suggestion
REC_COMMAND = re.compile(r'^\s*(`|\$\()\s*(.*)\s*([`)])$')
```

Reply:
This was exactly what was used before, but happy to change it.